spec/helpers/skins_helper_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe SkinsHelper do
+  describe "#current_skin" do
+    before do
+      allow(helper).to receive(:current_user)
+      allow(helper).to receive(:logged_in_as_admin?).and_return(false)
+      allow(helper).to receive(:logged_in?).and_return(false)
+      admin_setting = AdminSetting.default
+      admin_setting.default_skin = Skin.default
+      admin_setting.save(validate: false)
+    end
+
+    context "when the parameters include a skin id" do
+      before do
+        params[:site_skin] = skin.id
+      end
+
+      context "and the skin is usable" do
```

Comment:
I'm not sure "usable" is a good word here, since we have a `usable` method for skins that checks if `unusable: false`. "The skin is applied" or something?